pom.xml

```diff
@@ -21,10 +21,10 @@
 
 		<jackson.version>2.18.2</jackson.version>
 		<cxf.version>4.1.0</cxf.version>
-		<spring.version>6.2.1</spring.version>
+		<spring.version>6.2.2</spring.version>
```

Comment:
The updated dependencies seem to help making the test stable for all combinations, including WildFly + Artemis which was previously failing.